backend/src/common/plugins/authorization/authorization.plugin.ts

```diff
@@ -0,0 +1,38 @@
+import fp from 'fastify-plugin';
+
+import {
+    ControllerHook,
+    ExceptionMessage,
+    HttpCode,
+    HttpError,
+} from '../../enums/enums.js';
+
+// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
?

Reply:
Resolved in [a3bc8e0](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/115/commits/a3bc8e0a39f9ee743fb9a130e4675efd7c782ed5)